our-work/collaborations/yajima.yml

```diff
@@ -0,0 +1,18 @@
+title: Proteomics RShiny App
```

Comment:
Move to `Sofwatware`

Reply:
Looks like this still needs to be moved to software?

Reply:
Or should this stay in collaborations?